app/components/why-input.jsx

```diff
@@ -1,44 +1,145 @@
-// https://github.com/EnCiv/civil-pursuit/issues/68
-
 'use strict'
-import React from 'react';
-import Point from './point';
-import PointInput from './point-input';
-import cx from 'classnames';
-import { createUseStyles } from 'react-jss';
+import React, { useState, useEffect } from 'react'
```

Comment:
please undo the change to why-input.  I'm guessing this was an experiment. `git checkout master app/components/why-input.jsx`